File: chapters/10_security_sandbox_pairing/v1_no_protection.py
from __future__ import annotations


def execute_without_guard(command: str) -> dict[str, str]:
    return {
        "decision": "allowed",
        "command": command,
        "note": "no protection enabled",
    }


def main() -> None:
    print(execute_without_guard("rm -rf /tmp/demo"))


if __name__ == "__main__":
    main()

File: chapters/10_security_sandbox_pairing/v1_no_protection.ts
// TypeScript mirror for Chapter 10: Security Sandbox Pairing (v1)
// Run with: npx ts-node v1_no_protection.ts

function executeWithoutGuard(command: string): { decision: string; command: string; note: string } {
  return {
    decision: "allowed",
    command,
    note: "no protection enabled",
  };
}

function main(): void {
  console.log(executeWithoutGuard("rm -rf /tmp/demo"));
}

main();
